bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeSystem.scala

```diff
@@ -18,4 +29,11 @@ object MagnitudeSystem {
   implicit val MagnitudeSystemEqual: Equal[MagnitudeSystem] =
     Equal.equalA
 
+
+  // ===== LEGACY JAVA SUPPORT =====
+  val allAsJava = new java.util.Vector[MagnitudeSystem](all)
+  val DEFAULT = default
+
+  def valueOf(s: String) = all.find(_.name == s).get
```

Comment:
This takes the place of the `valueOf()` method on the Java enum which also throws an exception if the value can not be found. This is only used in the `PIOXml` code. Probably I should move this there so this is clear.